client.go

```diff
@@ -38,6 +38,9 @@ type Client interface {
 	// Replicas returns the set of all replica IDs for the given partition.
 	Replicas(topic string, partitionID int32) ([]int32, error)
 
+	// Isr returns the set of in-sync replica IDs for the given partition.
+	Isr(topic string, partitionID int32) ([]int32, error)
```

Comment:
For a public API I would prefer this be a bit more explicit: `ReplicasInSync` or `InSyncReplicas` or something like that. It would also be nice if the docs explained (or linked to an explanation) of what it means to be in sync.